downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md

```diff
@@ -113,19 +113,75 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 10 (2201.10.0)
     ```
 
 #### OpenAPI tool
-
+-  Added support to generate a mock client for OpenAPI specifications (OAS) that include examples. To generate a mock client, use the `--mock` flag in the OpenAPI client generation CLI command.
+  ```
+  $ bal openapi -i <yml file> --mode client --mock
+  ```
```

Comment:
````suggestion

-  Added support to generate a mock client for OpenAPI specifications (OAS) that include examples. To generate a mock client, use the `--mock` flag in the OpenAPI client generation CLI command.

  ```
  $ bal openapi -i <yml file> --mode client --mock
  ```

````

Please fix new lines for all the points.

Reply:
Sorry few comments getting conflicts while resolvings , will add